packages/react/src/ArqueroDetailsList/util/index.ts

```diff
@@ -9,6 +9,16 @@ export function getValue(item: any, column?: IColumn): any {
 	return column?.fieldName && item[column.fieldName]
 }
 
+export function getDropdownValue(item: any, column?: IColumn): any {
+	const itens = column?.fieldName && item[column.fieldName]
+	return itens.map((value: any) => {
+		return {
+			key: value,
```

Comment:
I see React errors about duplicate keys - we may want to add the index to the key to ensure it stays unique